website/fetchSupporters.js

```diff
@@ -7,46 +7,46 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const fs = require('fs');
-const request = require('request');
+const fs = require('fs').promises;
+const {request} = require('graphql-request');
 const path = require('path');
 
-const REQUIRED_KEYS = ['fromAccount', 'tier', 'totalDonations'];
-
-request(
-  'https://rest.opencollective.com/v2/jest/orders/incoming/active?limit=1000',
-  (err, response, body) => {
-    if (err) console.error('Failed to fetch backers: ', err);
-
-    // Basic validation
-    const result = JSON.parse(body);
-    if (!result || !Array.isArray(result.nodes)) {
-      throw new Error('backer info is not an array');
-    }
-
-    const backers = result.nodes;
-
-    for (const item of backers) {
```

Comment:
since we request via gql I don't think this is needed - the contract is guaranteed by the transport